teekkari.py

```diff
@@ -52,3 +58,9 @@ def messageHandler(self, bot, update):
                 self.handleHakemus(bot, update)
             elif 'diagnoosi' in msg.text.lower():
                 self.getDiagnoosi(bot, update)
+            elif 'halo' in msg.text.lower():
```

Comment:
`elif 'halo' in msg.text.lower():` is pointless. You can move the regex over here.